queries.sql

```diff
@@ -8,3 +8,36 @@ SELECT  name, escape_attempts from animals WHERE   weight_kg > 10.5;
 SELECT * from animals WHERE   neutered = true;
 SELECT * from animals WHERE   name <> 'Gabumon';
 SELECT * from animals WHERE  weight_kg >= 10.4 and weight_kg  <= 17.3;
+
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species is null;
+UPDATE animals SET species = 'pokemon' WHERE species is null;
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+UPDATE animals SET weight_kg = weight_kg * -1;
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+COMMIT;
```

Comment:
- [ ] This commit seems out of place, as it should not be there at all. Looking at the query questions,

Reply:
Opps, thank you dear for pointing that out! @bobb-Rob